### PR TITLE
create batch for testoutcome and featureLabel

### PR DESCRIPTION
**Pull request by heatherzheng**

### CODE REVIEW

File: policybot/mgrs/syncmgr/mgr.go [hunk 1]
```diff
@@ -1277,24 +1277,32 @@ func (ss *syncState) handlePostSubmitTestResults() error {
...
-			err = ss.mgr.store.WriteFeatureLabel(ss.ctx, FeatureLabels)
-			if err != nil {
-				return fmt.Errorf("unable to write FeatureLabel: %v", err)
+			testOutcomeFeatureLabel := &storage.TestOutcomeFeatureLabel{}
+			testOutcomeFeatureLabel.TestOutcome = TestOutcomes
+			testOutcomeFeatureLabel.FeatureLabel = FeatureLabels
+			return testOutcomeFeatureLabel, nil
+		}).Batch(5).To(func(input interface{}) error {
+			for _, i := range input.([]interface{}) {
+				err := ss.mgr.store.WriteTestOutcome(ss.ctx, i.(*storage.TestOutcomeFeatureLabel).TestOutcome)
```
Comment (therealmitchconnors) [inline policybot/mgrs/syncmgr/mgr.go:1294]: This is still writing all testoutcomes for one suite in a single operation, and that number may be very high (1000+) for a single suite... have you run into any errors running this locally?

Comment (heatherzheng) [reply policybot/mgrs/syncmgr/mgr.go:1294]: I deleted all batches locally and change it to WithParallelism. It's still running and haven't got any error yet.

Comment (heatherzheng) [reply policybot/mgrs/syncmgr/mgr.go:1294]: I changed the batch of PostSubmitTestOutcomes to 5 and batch of TestOutcomes and feature lables to 1. It works, and my cloudtop still is running.

Comment (therealmitchconnors) [reply policybot/mgrs/syncmgr/mgr.go:1294]: Ok.  We may need to watch for future failures, and divide into smaller batches again, but this seems good enough to me.

**Pull request merged**